src/_guides/language/effective-dart/usage.md

```diff
@@ -1013,6 +1013,37 @@ View(Style style, List children)
 an error is reported.
 </div>
 
+
+### DON'T use `new`.
+
+Dart 2 makes the `new` keyword optional. Even in Dart 1, its meaning was never
+clear because factory constructors mean a `new` invocation may still not
+actually return a new object.
+
+The language still permits `new` in order to make migration less painful, but
+consider it deprecated and remove it from your code.
+
+
+### DON'T use `const` when not needed.
+
+In contexts where only constant expressions are allowed, the `const` keyword is
+implicit, doesn't need to be written, and shouldn't. Those contexts are:
```

Comment:
This little in-sentence list isn't parallel and stopped me cold (shouldn't what? I knew after a second, but it still stopped me). Also, the intro to the bulleted list should be a sentence. How about:

...is implicit. Don't use `const` in the following constant contexts: